.travis.yml

```diff
@@ -1,19 +1,22 @@
 language: go
-dist: xenial
+dist: bionic
 notifications:
   email: true
 go:
-  - 1.11.x
+  - 1.14.x
```

Comment:
1.14 is untested with the container ecosystem. 1.13 is safer.

Reply:
Ok, I can downgrade to 1.13. Where is this information about safety for Go published? How do I track when a Go version is safe in the container ecosystem?

Reply:
https://github.com/moby/buildkit/pull/1377

Reply:
@AkihiroSuda this is now updated.